### PR TITLE
YR-[CIAC-6129]-fix-8-commands-to-work-with-a-single-FW

### PR DESCRIPTION
**Pull request by RosenbergYehuda**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6129

## Description
Fixed a bug in PAN_OS that appeared to cause 8 commands to work only when given a device group and not when given a single firewall.
After investigation it turned out that 7 commands worked fine as is, and only one command had to be fixed.
the command I fixed is :
 pan-os-apply-dns-signature-policy

The commands I verified work without any need for a fix:
 pan-os-apply-security-profile
 pan-os-commit-status
 pan-os-commit
 pan-os-list-edls
 pan-os-list-rules
 pan-os-refresh-edl
pan-os-create-rule
pan-os-edit-rule

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/PAN-OS/ReleaseNotes/1_16_21.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Added support for the **pan-os-apply-dns-signature-policy** command, to support a single firewall.
```
Comment (ShacharKidor) [inline Packs/PAN-OS/ReleaseNotes/1_16_21.md:6]:
```suggestion
- Fixed an issue where the ***pan-os-apply-dns-signature-policy*** command, didn't support a single firewall.
```

**Review by ShacharKidor — APPROVED** [review]
Looks good!

@RosenbergYehuda  - I've added a suggestion for the RN wordings.

@GuyAfik  - Would appreciate your review here.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 2]
```diff
@@ -7139,6 +7141,10 @@ def apply_dns_signature_policy_command(args: dict) -> CommandResults:
                    f'</lists>'
                    f'</botnet-domains>'
     }
+    if VSYS:  # if it's a firewall instanse, modify the xpath param
```
Comment (ShacharKidor) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:7144]:
```suggestion
    if VSYS:  # if it's a firewall instance, modify the xpath param
```

**Review by GuyAfik — APPROVED** [review]
@RosenbergYehuda nicely done!

**Pull request closed without merging**